docs/server/source/production-deployment-template-tendermint/workflow.rst

```diff
@@ -0,0 +1,138 @@
+Overview
+========
+
+This page summarizes the steps *we* go through
+to set up a production BigchainDB + Tendermint cluster.
+We are constantly improving them.
+You can modify them to suit your needs.
+
+.. Note::
+    With our BigchainDB + Tendermint deployment model, we use standalone MongoDB
+    (without Replica Set), BFT replication is handled by Tendermint.
+
+
+1. Set Up a Self-Signed Certificate Authority
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use SSL/TLS and self-signed certificates
+for MongoDB authentication (and message encryption).
+The certificates are signed by the organization managing the cluster.
+If your organization already has a process
+for signing certificates
+(i.e. an internal self-signed certificate authority [CA]),
+then you can skip this step.
+Otherwise, your organization must
+:ref:`set up its own self-signed certificate authority <how-to-set-up-a-self-signed-certificate-authority>`.
+
+
+.. _register-a-domain-and-get-an-ssl-certificate-for-it-tmt:
+
+2. Register a Domain and Get an SSL Certificate for It
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The BigchainDB APIs (HTTP API and WebSocket API) should be served using TLS,
+so the organization running the cluster
+should choose an FQDN for their API (e.g. api.organization-x.com),
+register the domain name,
+and buy an SSL/TLS certificate for the FQDN.
+
+.. _things-each-node-operator-must-do-tmt:
+
+Things Each Node Operator Must Do
+---------------------------------
+
+#. Name of the MongoDB instance (``mdb-instance-*``)
+#. Name of the BigchainDB instance (``bdb-instance-*``)
+#. Name of the NGINX instance (``ngx-http-instance-*`` or ``ngx-https-instance-*``)
+#. Name of the OpenResty instance (``openresty-instance-*``)
+#. Name of the MongoDB monitoring agent instance (``mdb-mon-instance-*``)
+#. Name of the Tendermint instance (``tendermint-instance-*``)
+
+
+☐ Generate two keys and corresponding certificate signing requests (CSRs):
+
+#. Client Certificate for BigchainDB Server to identify itself to MongoDB
+#. Client Certificate for MongoDB Monitoring Agent to identify itself to MongoDB
+
+Ask the managing organization to use its self-signed CA to sign those four CSRs.
```

Comment:
Here the node operator doesn't have to ask the cluster's managing organization to do the signing. Each node operator can just sign everything themselves using their locally-generated self-signed CA.

Reply:
+1

Reply:
:+1: